view/ViewableData.php

```diff
@@ -273,13 +273,11 @@ public function castingClass($field) {
 	 * @return string 'xml'|'raw'
 	 */
 	public function escapeTypeForField($field) {
-		if(!$class = $this->castingClass($field)) {
-			$class = self::$default_cast;
-		}
-		
+		$class = $this->castingClass($field) ?: $this->config()->get('default_cast');
```

Comment:
Everywhere else uses `Config::inst()->get('ViewableData', 'default_cast', Config::FIRST_SET);`, but that might be wrong as `$class` might end up being null